docs/disk-resizer.md

```diff
@@ -4,18 +4,17 @@
 
 Aliyun EBS support to expand size of disk, this feature will show how to resize ebs in kubernetes platform.
 
-*The CSI-Resizer is Alpha feature in k8s 1.14.*
+*The CSI-Resizer is Alpha feature in k8s 1.14, Beta feature in k8s 1.16+*
```

Comment:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/

GA in 1.24

Reply:
okay